code/online_challenges/src/hackerrank/absolute_permutation/absolute_permutation.cpp

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+
+int main(){
+    int t;
+    std::cin >> t;
+    for(int a0 = 0; a0 < t; ++a0){
```

Comment:
Could you please match our C++ style guide for all of your files? It is located in `cosmos/guides/coding_style/C++`.